File: ECOMEP.WebApi/MyCockpitView.WebApi/WFTaskModule/Dtos/AssessmentDto.cs
using MyCockpitView.WebApi.ContactModule.Dtos;
using MyCockpitView.CoreModule;

namespace MyCockpitView.WebApi.WFTaskModule.Dtos;

public class AssessmentDto : BaseEntityDto
{

    public int? EntityID { get; set; }
    public string? Entity { get; set; }
    public string? EntityTitle { get; set; }
    public int ContactID { get; set; }
    public virtual ContactListDto? Contact { get; set; }


    public string? TaskTitle { get; set; }
    public string? Category { get; set; }

    public decimal Points { get; set; }

    public decimal ScoredPoints { get; set; }

    public string? Comment { get; set; }

    public int? WFTaskID { get; set; }

}

File: ECOMEP.Angular/src/app/shared/models/assessment-dto.ts
import { Contact } from "src/app/contact/models/contact";
import { BaseEntity } from "src/app/shared/models/base-entity.model";

export class AssessmentDto extends BaseEntity
{
  contactID: number = 0;
  contact: Contact = new Contact();
  taskTitle!: string;
  category!: string;
  points: number = 0;
  scoredPoints: number = 4;
  wfTaskID: number = 0;
}
